code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -50,13 +50,24 @@
 /datum/chemical_reaction/plasmasolidification
 	name = "Solid Plasma"
 	id = "solidplasma"
-	required_reagents = list("iron" = 5, "frostoil" = 5, "plasma" = 20)
+	required_reagents = list("frostoil" = 5, "plasma" = 20)
```

Comment:
@phil235 mob_react is backwards tho ^^
mob_react = 1 means that it will NOT react inside of a mob. This is set explicitly for the reaction.
mob_react = 0 means that it WILL react inside of a mob. 0 is the default.